php-frontend/src/main/java/org/sonar/php/parser/TreeFactory.java

```diff
@@ -0,0 +1,1100 @@
+      eqToken = eqAndInitValue.get().first();
+      initValue = eqAndInitValue.get().second();
+    }
+    VariableIdentifierTree varIdentifier = new VariableIdentifierTreeImpl(new IdentifierTreeImpl(identifier));
+    return new ParameterTreeImpl(classType.orNull(), ampersand.orNull(), ellipsis.orNull(), varIdentifier, eqToken, initValue);
+  }
+
+  /**
+   * [ END ] Declarations
+   */
+
+  
+  /**
+   * [ START ] Statement
+   */
+
+  public GlobalStatementTree globalStatement(
+      InternalSyntaxToken globalToken, VariableTree variable,
+      Optional<List<Tuple<InternalSyntaxToken, VariableTree>>> variableRest, InternalSyntaxToken eosToken
+  ) {
+    return new GlobalStatementTreeImpl(
+        globalToken,
+        separatedList(variable, variableRest),
+        eosToken
+    );
+  }
+
+  public UseStatementTree useStatement(
+      InternalSyntaxToken useToken,
+      Optional<InternalSyntaxToken> useTypeToken,
+      UseClauseTree firstDeclaration,
+      Optional<List<Tuple<InternalSyntaxToken, UseClauseTree>>> additionalDeclarations,
+      InternalSyntaxToken eosToken
+  ) {
+    SeparatedList<UseClauseTree> declarations = separatedList(firstDeclaration, additionalDeclarations);
+    return new UseStatementTreeImpl(useToken, useTypeToken.orNull(), declarations, eosToken);
+  }
+
+  public ReturnStatementTree returnStatement(InternalSyntaxToken returnToken, Optional<ExpressionTree> expression, InternalSyntaxToken eos) {
+    return new ReturnStatementTreeImpl(returnToken, expression.orNull(), eos);
+  }
+
+  public ContinueStatementTree continueStatement(InternalSyntaxToken continueToken, Optional<ExpressionTree> expression, InternalSyntaxToken eos) {
+    return new ContinueStatementTreeImpl(continueToken, expression.orNull(), eos);
+  }
+
+  public BreakStatementTree breakStatement(InternalSyntaxToken breakToken, Optional<ExpressionTree> expression, InternalSyntaxToken eos) {
+    return new BreakStatementTreeImpl(breakToken, expression.orNull(), eos);
+  }
+
+  public BlockTree block(InternalSyntaxToken lbrace, Optional<List<StatementTree>> statements, InternalSyntaxToken rbrace) {
+    return new BlockTreeImpl(lbrace, optionalList(statements), rbrace);
+  }
+
+  public GotoStatementTree gotoStatement(InternalSyntaxToken gotoToken, InternalSyntaxToken identifier, InternalSyntaxToken eos) {
+    return new GotoStatementTreeImpl(gotoToken, new IdentifierTreeImpl(identifier), eos);
+  }
+
+  public ExpressionStatementTree expressionStatement(ExpressionTree expression, InternalSyntaxToken eos) {
+    return new ExpressionStatementTreeImpl(expression, eos);
+  }
+
+  public LabelTree label(InternalSyntaxToken identifier, InternalSyntaxToken colon) {
+    return new LabelTreeImpl(new IdentifierTreeImpl(identifier), colon);
+  }
+
+
+  public TryStatementTree tryStatement(InternalSyntaxToken tryToken, BlockTree blockTree, Optional<List<CatchBlockTree>> catchBlocks, Optional<Tuple<InternalSyntaxToken, BlockTree>> finallyBlock) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 197 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00103)